src/modules/product/product.controller.js

```
@@ -1,15 +1,21 @@
+//   updateProductService,
+//   deleteProductService,
+//   getAllCategoriesService,
+//   getCategoryByIdService,
+//   createCategoryService,
+//   updateCategoryService,
+//   deleteCategoryService,
+//   getProductCountService
+// } from "./product.service.js";
+import ProductService from "./product.service.js";
+import cloudinaryService from "../../common/services/cloudinary.service.js";
+import fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)